src/windows/config/index.html

```diff
@@ -15,6 +15,9 @@
       </div>
       <div class="mobbers">
         <h1>Mobbers</h1>
+        <div class="mobberActions">

```

Comment:
Hmm... isn't adding a mobber also a "mobber action?" Not sure if this is the right name given the placement, but it's a minor quibble.

Reply:
Good point! I'm open to suggestions on a better name, will think about that one